src/Illuminate/Encryption/Encrypter.php

```diff
@@ -33,16 +34,23 @@ class Encrypter implements EncrypterContract
      * Create a new encrypter instance.
      *
      * @param  string  $key
-     * @param  string|null  $cipher
+     * @param  string  $cipher
      * @return void
      */
-    public function __construct($key, $cipher = null)
+    public function __construct($key, $cipher = 'AES-128-CBC')
     {
-        $this->key = (string) $key;
+        $len = mb_strlen($key = (string) $key, '8bit');
```

Comment:
@GrahamCampbell , I saw a commit from you adding `ext-mcrypt`, how come this not in composer.json?

https://github.com/laravel/framework/commits/5.0/src/Illuminate/Encryption/composer.json

Is it me or something weird is going on? :confused: